src/scripts/mobile-menu.ts

```
@@ -0,0 +1,25 @@
+
+    if (mobileMenuButton && mobileMenu && mobileMenuCloseButton) {
+        mobileMenuButton.addEventListener('click', () => {
+            mobileMenu.classList.remove('hidden');
+        });
+        mobileMenuCloseButton.addEventListener('click', () => {
+            mobileMenu.classList.add('hidden');
+        });
+        
+        // Optional: Close mobile menu if clicking outside of it (on the overlay)
+        // The overlay is the first direct child of #mobile-menu in the account template.
+        if (mobileMenu.firstElementChild && mobileMenu.firstElementChild.classList.contains('fixed', 'inset-0', 'bg-black')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)